app/_src/gateway/kong-manager/enable.md

```diff
@@ -19,6 +19,21 @@ or hybrid mode), you can enable {{site.base_gateway}}'s graphical user interface
   admin_gui_path = /manager
   ```
 > Make sure that each domain has proper DNS records and that the Kong instance is accessible from all specified domains.
+
+{:.important}
+> **Important**: If your setup involves multiple domains or subdomains, it’s generally recommended to remove the `cookie_domain` that setting in the `admin_gui_session_conf` or `admin_gui_auth_conf`.
+> When `cookie_domain` is not specified, cookies are set for the origin domain by default. This allows the browser to manage cookies correctly for each domain independently, avoiding conflicts or scope issues. For example:
```

Comment:
I realized that the cookies will be set for `admin_gui_api_url` if it's specified but `cookie_domain` is not, and `admin_gui_api_url` is not necessary to be affiliated with `admin_gui_url`. To make it more accurate:

```suggestion
> When `cookie_domain` is not specified, cookies are set for the domain initiated the request, if `admin_gui_api_url` is not specified. This allows the browser to manage cookies correctly for each domain independently, avoiding conflicts or scope issues. For example:
```